campaigns/20150429-sign-letter.html

```diff
@@ -11,16 +11,19 @@
 ---
 <link href='http://fonts.googleapis.com/css?family=Roboto+Slab:400,700' rel='stylesheet' type='text/css'>
 <style>
-  .jumbotron{
+/*  .jumbotron{
     margin-top: -45px;
     padding:0;
     position:relative;
     overflow: hidden;
-  }
+    @include phone{
+      margin-top: 0px;
+    }
+  }*/
   .jumbotron>.container {
     padding-top:30px;
     padding-bottom:15px;
-    background: url("/images/campaigns/20150429-hero-bg.jpg") 0 -0 no-repeat;
+    // background: url("/images/campaigns/20150429-hero-bg.jpg") 0 -0 no-repeat;
```

Comment:
This is all stuff that happened as i tried to get this page to work from the merge that was from 24 hours ago.  Please feel free to turn this back on when you make sure the right content is on the page.  Again Because the merge I created was from yesterday, I had these convicts just because so much had changed.